### ISSUE TITLE
Unable to find the gold annotations for test set

### ISSUE THREAD

**Issue by nurakib** [THU-KEG/MAVEN-ERE#4]
I downloaded the dataset from google drive, and the test split has no relations mentioned in the test jsonlines file (train and validation set do have it). Where do I find the gold event-to-event relations?

**Comment by Bakser** [comment]
Hi, we decided to keep the test set annotation undisclosed to maintain a fair and consistent evaluation environment. If you want to get the results on the test set, you can submit your predictions to CodaLab following the [instructions in README](https://github.com/THU-KEG/MAVEN-ERE#get-test-results-from-codalab).
